### PR TITLE
CIV-5973 Convert ga ccd to nodejs pipeline

### PR DESCRIPTION
**Pull request by karthick-mohanasundaram-HMCTS**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-5973

### Change description ###

Converting ga ccd to nodejs pipeline and add unit tests

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by karthick-mohanasundaram-HMCTS** [comment]
API
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
Smoke test

image

File: build.gradle [hunk 1]
```diff
@@ -1,255 +1,39 @@
...
-  implementation group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.5'
-  constraints {
-    implementation('org.apache.commons:commons-compress:1.21') {
-      because 'CVE-2021-35515, CVE-2021-35516, CVE-2021-35517, CVE-2021-36090'
-    }
-  }
+  implementation group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.8'
+  implementation group: 'org.slf4j', name: 'slf4j-api', version: '2.0.3'
 
   testImplementation(platform('org.junit:junit-bom:5.8.2'))
```
Comment (vasudevganesanhmcts) [inline build.gradle:34]: we dont need this and should be removed

File: build.gradle [hunk 2]
```diff
@@ -1,255 +1,39 @@
...
+task highLevelDataSetup(type: JavaExec) {
+  main = "uk.gov.hmcts.reform.civil.HighLevelDataSetupApp"
+  classpath += sourceSets.main.runtimeClasspath
+  jvmArgs = ['--add-opens=java.base/java.lang.reflect=ALL-UNNAMED']
 }
 
 dependencies {
-  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web'
-  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-actuator'
   implementation group: 'org.springframework.boot', name: 'spring-boot-starter-aop'
```
Comment (vasudevganesanhmcts) [inline build.gradle:28]: should be removed

File: build.gradle (hunk 1 above)
Comment (karthick-mohanasundaram-HMCTS) [reply build.gradle:34]: will look into it. Thanks

**Review by vasudevganesanhmcts — APPROVED** [review]

**Pull request merged**